cmd/minishift/cmd/start.go

```diff
@@ -409,23 +433,23 @@ func determineIsoUrl(iso string) string {
 	return iso
 }
 
-// initStartFlags creates the CLI flags which needs to be passed on to 'libmachine'
-func initStartFlags() *flag.FlagSet {
-	startFlagSet := flag.NewFlagSet(commandName, flag.ContinueOnError)
-
-	startFlagSet.String(configCmd.ISOUrl.Name, configCmd.B2dIsoAlias, "Location of the minishift ISO. Can be an URL, file URI or one of the following short names: [b2d centos].")
-	startFlagSet.String(configCmd.VmDriver.Name, constants.DefaultVMDriver, fmt.Sprintf("The driver to use for the Minishift VM. Possible values: %v", constants.SupportedVMDrivers))
-	startFlagSet.Int(configCmd.Memory.Name, constants.DefaultMemory, "Amount of RAM to allocate to the Minishift VM.")
-	startFlagSet.Int(configCmd.CPUs.Name, constants.DefaultCPUS, "Number of CPU cores to allocate to the Minishift VM.")
-	startFlagSet.String(configCmd.DiskSize.Name, constants.DefaultDiskSize, "Disk size to allocate to the Minishift VM. Use the format <size><unit>, where unit = b, k, m or g.")
-	startFlagSet.String(configCmd.HostOnlyCIDR.Name, "192.168.99.1/24", "The CIDR to be used for the minishift VM. (Only supported with VirtualBox driver.)")
-	startFlagSet.StringArrayVar(&dockerEnv, configCmd.DockerEnv.Name, nil, "Environment variables to pass to the Docker daemon. Use the format <key>=<value>.")
-	startFlagSet.StringSliceVar(&dockerEngineOpt, configCmd.DockerEngineOpt.Name, nil, "Specify arbitrary flags to pass to the Docker daemon in the form <flag>=<value>.")
-	startFlagSet.StringSliceVar(&insecureRegistry, configCmd.InsecureRegistry.Name, []string{defaultInsecureRegistry}, "Non-secure Docker registries to pass to the Docker daemon.")
-	startFlagSet.StringSliceVar(&registryMirror, configCmd.RegistryMirror.Name, nil, "Registry mirrors to pass to the Docker daemon.")
-	startFlagSet.AddFlag(cmdutil.AddOnEnvFlag)
-
-	return startFlagSet
+// initconfigCmd creates the CLI flags which needs to be passed on to 'libmachine'
+func initconfigCmd() *flag.FlagSet {
+	configCmdet := flag.NewFlagSet(commandName, flag.ContinueOnError)
```

Comment:
Is this related to the merge conflict or what does _configCmdet_ mean. It feels like an over eager find and replace